scripts/assets/zip/README.md

```diff
@@ -0,0 +1,13 @@
+# CKEditor 5 sample
+
+Thank you for downloading the sample. To run it, start a web server (`nginx`, `caddy` or any other) that will serve the file.
+
+Want to test more features? Get started with CKEditor 5 Builder 🚀
+
+With [CKEditor’s interactive builder](https://ckeditor.com/ckeditor-5/builder/), select:
+
+* The features you need.
+* Your preferred framework (React, Angular, Vue, or Vanilla JS).
+* Your preferred distribution method (CDN or npm).
```

Comment:
@Witoso, we miss the preset selection. Users can select the predefined configuration that can be adjusted to their needs or start from scratch by selecting the editor type only.